src/package/components/InstructorStats/index.js

```diff
@@ -0,0 +1,63 @@
+import React, {PropTypes} from 'react'
+import pluralize from 'pluralize'
+import {map, uniqBy} from 'lodash'
+import {hasUnlockedPublished, hasUnlockedCoursePublished} from 'package/utils/instructorMilestones'
+import createLessonsUrlWithParams from 'package/utils/createLessonsUrlWithParams'
+import Maybe from 'package/components/Maybe'
+import Card from 'package/components/Card'
+import Request from 'package/components/Request'
+import InstructorStat from './components/InstructorStat'
+
+const InstructorStats = ({instructor}) => {
+
+  if(!hasUnlockedPublished(instructor.published_lessons)) {
+    return null
+  }
+
+  return (
+    <Card>
+      <div className='pa5'>
+        <Request
+          url={createLessonsUrlWithParams({
+            lessonsUrl: instructor.lessons_url
+          })}
+        >
+          {({data}) => (
+            <InstructorStat
+              count={instructor.published_lessons}
+              label={pluralize('Lesson', instructor.published_lessons)}
+              graphics={uniqBy(map(data, lesson => ({
+                name: lesson.technology.name,
+                graphicUrl: lesson.technology.logo_http_url,
+                httpUrl: lesson.technology.http_url,
+              })), technology => technology.name)}
+            />
+          )}
+        </Request>
+        <Maybe condition={hasUnlockedCoursePublished(instructor.published_courses)}>
+          <div className='mt4'>
+            <Request url={`${process.env.REACT_APP_EGGHEAD_BASE_URL}/api/v1/series?instructor_id=${instructor.id}`}>
```

Comment:
Thanks! It must not be in staging yet because I didn't see it and created this story to address this: https://github.com/eggheadio/egghead-ui/issues/113
Will deal with this separately because I still need the `root.technologies_url` as well to completely remove link building from `/package/*` ^